lib/splitclient-rb/sse/workers/splits_worker.rb

```diff
@@ -44,15 +44,18 @@ def update_feature_flag(notification)
 
           if @feature_flags_repository.get_change_number == notification.data['pcn']
             begin
-              @feature_flags_repository.add_split(
-                JSON.parse(
-                  SplitIoClient::Helpers::DecryptionHelper.get_encoded_definition(
-                    notification.data['c'],
-                    notification.data['d']
-                  ),
-                  symbolize_names: true
-                )
+              new_split = JSON.parse(
```

Comment:
i know that we already spoke about this, but i prefer to prevent to throw an exception and check before if you have the definition